src/modules/haskell.rs

```diff
@@ -0,0 +1,60 @@
+use ansi_term::Color;
+use std::process::Command;
+
+use super::{Context, Module};
+
+/// Creates a module with the current Haskell Stack version
+///
+/// Will display the Haskell version if any of the following criteria are met:
+///     - Current directory contains a `stack.yaml` file
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let is_haskell_project = context
+        .try_begin_scan()?
+        .set_files(&["stack.yaml"])
+        .is_match();
+
+    if !is_haskell_project {
+        return None;
+    }
+
+    match get_haskell_version() {
+        Some(haskell_version) => {
+            const HASKELL_CHAR: &str = "λ ";
+
+            let mut module = context.new_module("haskell");
+            let module_style = module
+                .config_value_style("style")
+                .unwrap_or_else(|| Color::Red.bold());
+            module.set_style(module_style);
+
+            let formatted_version = format_haskell_version(&haskell_version)?;
+            module.new_segment("symbol", HASKELL_CHAR);
```

Comment:
Is it really necessary to repeat all the default values from `src/configs/haskell.rs`?

Reply:
It seems that you have mixed the old config system with the new one. Check https://github.com/starship/starship/blob/master/src/modules/ruby.rs#L26-L31 about how you can write the code.